zerver/lib/events.py

```diff
@@ -441,6 +441,8 @@ def name(sub):
     elif event['type'] == "update_display_settings":
         if event['setting_name'] == "twenty_four_hour_time":
             state['twenty_four_hour_time'] = event["setting"]
+        if event['setting_name'] == "high_contrast_mode":
+            state['high_contrast_mode'] = event["setting"]
```

Comment:
Based on the context, you should have rebased onto master before writing this code.  

Also, you should be using the `property_types` feature to have this (and several other components) of these changes be handled automatically for you.  See zulip.readthedocs.io/en/latest/new-feature-tutorial.html for some notes on how that works.